lua/astrocommunity/completion/avante-nvim/init.lua

```diff
@@ -0,0 +1,97 @@
+return {
+  "yetone/avante.nvim",
+  build = ":AvanteBuild",
+  cmd = {
+    "AvanteAsk",
+    "AvanteBuild",
+    "AvanteConflictChooseAllTheirs",
+    "AvanteConflictChooseBase",
+    "AvanteConflictChooseBoth",
+    "AvanteConflictChooseCursor",
+    "AvanteConflictChooseNone",
+    "AvanteConflictChooseOurs",
+    "AvanteConflictChooseTheirs",
+    "AvanteConflictListQf",
+    "AvanteConflictNextConflict",
+    "AvanteConflictPrevConflict",
+    "AvanteEdit",
+    "AvanteRefresh",
+    "AvanteSwitchProvider",
+  },
```

Comment:
```suggestion
```
This can be removed if `lazy = false` is added

Reply:
whats the reason behind `lazy = false` ? it works fine without it.. and it imho it should follow the user config  and not force loading on startup (it will load a lot deps as well)

Reply:
> whats the reason behind `lazy = false` ? it works fine without it.. and it imho it should follow the user config and not force loading on startup (it will load a lot deps as well)

The docs dosent have lazy = true, they recommend lazy = false

Reply:
hey @luxus and @Uzaaft, i've been testing the plugin with the current configuration (leaving out `lazy = false` and define `cmd`s). loading on commands can reduce about 20ms start time at most on my end and it's working. if the speed matters i believe it's ready to go as it is.

Reply:
I'm going to defer that decision to luxux. but this can break in the future.

Reply:
@Uzaaft thank you for your help these days.